test/unit/domain/settlement/index.test.js~8af0011eba793e5abd3e561c2a35c8dd007f523e

```diff
@@ -0,0 +1,27 @@
+/*****
```

Comment:
this seems to be a merge conflict backup file. whats with the file name?

Reply:
There was a conflict but it was resolved. Such filenames do not exist in the file tree.